integration/005_lnurlw-race.jmx

```diff
@@ -2154,7 +2154,7 @@ if (resp.length !== parallelWithdrawUsers) {
   var withdrawOkCount = 0;
 
   for (var i = 0; i &lt; resp.length; i++) {
-    if (resp[i].pending === false) {
+    if (resp[i].status === &quot;success&quot;) {
```

Comment:
lol :)

Reply:
:dagger:

Reply:
extracted JS assertions to separate file.
see https://github.com/lnbits/lnbits-extensions/pull/365